### ISSUE TITLE
import statements

### ISSUE THREAD

**Issue by verbman** [ServiceInnovationLab/openfisca-aotearoa#80]
Hi there!

I really enjoy OpenFisca, but I recently encountered an issue.

### Here is what I did:
I used the default import statement:
`from openfisca_core.model_api import *`

### Here is what I expected to happen:
It worked but I felt like it's not best practice yet it's current practice and think we should have a conversation about it.

### Here is what I propose we do:
from openfisca_core.model_api import MONTH, YEAR

## Context

I identify more as a:
- Developer _(I create tools that use the existing OpenFisca code)_.

**Comment by Br3nda** [comment]
agreed. we might be able to convince the flake8 linter to fail these
